mturk/tasks.py

```diff
@@ -1,28 +1,41 @@
-from crowdsourcing.models import Project
+from crowdsourcing.models import Project, TaskWorker, Task
 from csp.celery import app as celery_app
-from csp.settings import SITE_HOST
+from csp.settings import SITE_HOST, AWS_DAEMO_KEY
 from mturk.interface import MTurkProvider
 from mturk.models import MTurkHIT
+from django.db.models import Q
+from django.contrib.auth.models import User
+from crowdsourcing.crypto import AESUtil
 
 
 @celery_app.task
 def mturk_publish():
-    provider = MTurkProvider(SITE_HOST)
-    projects = Project.objects.filter(deleted=False, min_rating__lt=0.61)
+    projects = Project.objects.filter(~Q(owner__profile__user__mturk_account=None), deleted=False, min_rating__lt=0.61,
```

Comment:
this is a global value anything greater than 0 and lower than 1 works (magic random choice to puzzle people lol)